src/components/RecordCount.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 export function RecordCount(props) {
   return (
-    <div>
+    <div className="recordCount">
```

Comment:
It's a good idea to give each component a root class name matching the component name.